tools/postinstall

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+pod_install() (
+  [[ "$OSTYPE" == "darwin"* ]] || { exit 0; };
+  hash pod 2>/dev/null || {
+    echo >&2 "It looks like you haven't installed CocoaPods. Please do so, following the instructions at https://reactnative.dev/docs/0.60/getting-started, at 'React Native CLI Quickstart' > 'Xcode & CocoaPods', and run `yarn` again."
```

Comment:
The `` `yarn` `` inside the double-quoted string here is a command substitution that will actually run `yarn`.

Reply:
Oh, nice. Is that still the case if I use a "here-document", e.g.:

```bash
        cat >&2 <<EOF
The postinstall script (see scripts.postinstall in package.json)
requires CocoaPods. Please run:

  sudo gem install cocoapods

as instructed at https://reactnative.dev/docs/0.60/getting-started,
under 'React Native CLI Quickstart' > 'Xcode & CocoaPods'.

Then, run `yarn` again.
EOF
```

and, if it's safe to include the backticks, is that a desired bit of formatting that we should keep?

Reply:
Aha! ShellCheck doesn't want me to use the backticks:

```
Line 19:
Then, run `yarn` again.
          ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean: (apply this, apply all SC2006)
Then, run $(yarn) again.
```

Reply:
Maybe I'll put it on its own line, the same as `sudo gem install cocoapods`, above.

Reply:
A here-document `<<word` does parameter expansion, command substitution, and arithmetic expansion. A here-document `<<'word'` does not. (Documentation for [POSIX shell](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_04) and [Bash](https://www.gnu.org/software/bash/manual/html_node/Redirections.html#Here-Documents).)